scenarioo-client/app/manage/comparisons/comparisonStatusMapper.service.js

```diff
@@ -0,0 +1,41 @@
+/* scenarioo-client
+ * Copyright (C) 2014, scenarioo.org Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+angular.module('scenarioo.services').factory('ComparisonStatusMapperService', function () {
+
+    var styleClassesForComparisonStatus = {
+        'SUCCESS': 'label-success',
+        'FAILED': 'label-danger',
+        'UNPROCESSED': 'label-default',
+        'QUEUED_FOR_PROCESSING': 'label-info',
+        'PROCESSING': 'label-primary'
```

Comment:
although I think we have too much states here - I think we can keep it - because once we go for queued processing of diffs this might indeed make sense. Thanks!